client/src/utils/FetchUserData.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable consistent-return */
+import Cookies from 'js-cookie';
+import loginAuth from './loginAuth';
+import URLpath from './URLpath';
+
+export async function fetchLogin(body) {
+  try {
+    const res = await fetch(URLpath('login'), {
```

Comment:
I do like how this is one of the few places we are using fetch, great job on that. Id say lets take it a step further, make its own utils file where `fetch` is only used there. That way if need to use something else, we only have one place to change it. Does that make sense?